### PR TITLE
Add view semesters

### PR DESCRIPTION
**Pull request by SkaeX**
- Add only view to semesters.

To do
- [x] Add question sets to view semesters view.
- [x] Add current semester tag to view semesters view.

### CODE REVIEW

**Comment by BrainMaestro** [comment]
Nice, but a semester also has question sets which are not displayed in the table. Also the page should show which semester is the current one.

File: resources/assets/js/components/shared/table.vue [hunk 1]
```diff
@@ -63,6 +63,10 @@
                                 item[key] = value.data.map(questionSet => questionSet.name);
                                 break;
 
+                            case 'current_semester':
```
Comment (BrainMaestro) [inline resources/assets/js/components/shared/table.vue]: Is this necessary? Looks the same as the default case

Comment (SkaeX) [inline resources/assets/js/components/shared/table.vue]: I wanted `String` not `Boolean`. Do you have a better way?

**Comment by BrainMaestro** [comment]
Sweet! What do we do about the empty space though?

**Comment by BrainMaestro** [comment]
Approved

**Comment by SkaeX** [comment]
We have some options:
- Leave it as it is 😆  (Easy & lazy option).
- Resize the table.
- Expand the tags to fill up the space (I discourage).

@BrainMaestro you can add more. I'll add more myself if I think of any.

**Comment by BrainMaestro** [comment]
If we resize the table, then the search bar will look out of place as well. 
Tricky.

**Pull request merged**